app/views/user_mailer/activation_limit_reached.html.erb

```diff
@@ -0,0 +1,49 @@
+<%#-- copyright
+OpenProject is a project management system.
+Copyright (C) 2012-2017 the OpenProject Foundation (OPF)
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2017 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See doc/COPYRIGHT.rdoc for more details.
+
+++#%>
+
+<p>
+    <% mail_link = "<a href=\"mailto:#{@email}\">#{@email}</a>" %>
+    <% host = Setting.protocol + "://" + Setting.host_name %>
+    <% host_link = "<a href=\"#{host}\">#{Setting.host_name}</a>" %>
+    <%= t("mail_user_activation_limit_reached.message", email: mail_link, host: host_link).html_safe %>
+</p>
+
+<p>
+    <span><%= t("mail_user_activation_limit_reached.steps.label") %></span>
+    <ul>
+        <li>
+            <%= t("mail_user_activation_limit_reached.steps.a") %>
+            <a href="<%= OpenProject::Enterprise.upgrade_url %>">here</a>
```

Comment:
missing I18n for `here` and while you are at it, it would probably make sense to include the anchor tag in the `steps.a` localization string so that the position of the link can be different in different languages